iam/certificate-acl/app/redis/RedisManager.js

```diff
@@ -27,28 +33,33 @@ class RedisManager {
    * Creates a Redis Manager
    *
    * @constructor
+   * @param {*} serviceStateManager instance of a ServiceStateManager
    * @returns Redis Manager Object
    */
-  constructor() {
+  constructor(serviceStateManager) {
```

Comment:
I'm not sure if it's really necessary, but remember that issue that we thought was a lib problem? #2094

https://github.com/dojot/dojot/blob/46aef2686aa4d4c1c7dacd2c16c84416597d5484/subscription-engine/kafka-ws/app/Redis/Utils.js#L15
https://github.com/dojot/dojot/blob/46aef2686aa4d4c1c7dacd2c16c84416597d5484/subscription-engine/kafka-ws/app/Redis/RedisManager.js#L73

Reply:
put this how TODO and I will approve the code

Reply:
:+1: